setup.py

```diff
@@ -21,15 +21,15 @@ def read_requirements(path: str):
 if version.endswith(("a", "b", "rc")):
     # append version identifier based on commit count
     try:
-        import subprocess
+        import subprocess  # noqa: S404
```

Comment:
Could probably disable S404 globally if it shows additional errors anyway when later using one of its methods

Reply:
I don't think its worth removing.